main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+
+	"github.com/jbydeley/beacon/config"
```

Comment:
I'm not super familiar with imports in Golang but when this merges to `the-heap`'s github account will the path for this need to be updated?